src/gluonts/model/deepstate/_network.py

```diff
@@ -138,7 +138,7 @@ def compute_lds(
             prior_mean=prior_mean,
             prior_cov=prior_cov,
             latent_dim=self.issm.latent_dim(),
-            output_dim=1,
+            output_dim=self.issm.output_dim(),
```

Comment:
There is one more place it is hardcoded: https://github.com/awslabs/gluon-ts/blob/29ab758c9ddebdbba39a12597a1d5fcaaf906a97/src/gluonts/model/deepstate/_network.py#L69